schema/vrs.yaml

```diff
@@ -249,62 +276,87 @@ definitions:
   # SequenceIntervals
 
   SequenceInterval:
+    description: >-
+      A SequenceInterval is intended to be compatible with that in Sequence Ontology
+      ([SO:0000001](http://www.sequenceontology.org/browser/current_svn/term/SO:0000001)),
+      with the exception that the GA4GH VRS SequenceInterval may be zero-width. The SO
+      definition is for an "extent greater than zero".
     oneOf:
+      - $ref: '#/definitions/NumericInterval'
       - $ref: '#/definitions/SimpleInterval'
-      - $ref: '#/definitions/NestedInterval'
     discriminator:
       propertyName: type
 
-  SimpleInterval:
+  NumericInterval:
```

Comment:
Assuming we go forward with the Numeric and Range data types being proposed in this PR, we should seriously consider whether to drop Interval and simply embed it SequenceLocation. It doesn't seem like it will ever be used elsewhere. Range is really the lower level general purpose data type that I think SimpleInterval was supposed to cover. Doing this change will simplify the Location data structures ChromosomeLocation with a start and end HumanCytoband type and SequenceLocation with a start and end Numeric|Range types. So this is an example of how these new changes give us back some simplicity in other areas of the model. IMO

Reply:
I think your suggestion makes sense. However, we cannot make breaking changes without a major version release and product review. Perhaps an alternative to the `NumericInterval` inside our current `SequenceLocation` is to instead create a simpler SequenceLocation-like class that covers this? Alternatively, run with `NumericInterval` and then refactor to simpler `SequenceLocation` in 2.0?

Reply:
if we go with the later suggestion then we should look forward to how we would refactor SequenceLocation in 2.0. I think we would still need to deprecate and create a new class for the new "simpler" version being suggested now. So, we could take that step now. Unless my assumption is wrong and we can simply reuse the same name for SequenceLocation and change it's structure in 2.0, in which case I would be fine with waiting. It is just a shame to not be able to change this now since there are very few implementations existing so far.

Reply:
Recast this as SequenceInterval and do away with abstract class of same name.